master/reference/public-cloud/azure.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Deploying Calico on Azure
+no_canonical: true
+---
+
+{{site.prodname}} in [Microsoft Azure][Azure]{:target="_blank"} supported in [policy-only][PolicyMode] mode.
+Networking part covered by Azure user-defined routes. {{site.prodname}} IPAM needs
```

Comment:
Can you insert an "is" just before supported? That would make the sentence more correct.

This sentence "Networking part covered by Azure user-defined routes." is problematic grammatically. I think you are definitely missing an article and a verb here. However, since this sentence just before this one says almost the same thing and you cover this topic so well just below, I think you can just delete this sentence.